server_monitoring/README.rst

```diff
@@ -0,0 +1,88 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+server monitoring
+==============
```

Comment:
You can add the same line on top to make the title more important (see https://github.com/OCA/maintainer-tools/pull/109)